src/pages/cli/graphql/override.mdx

```diff
@@ -76,6 +76,48 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
   resources.models["Todo"].modelDatasource.dynamoDbConfig['deltaSyncConfig']['baseTableTtl'] = '3600'
 }
 ```
+### Example - Configure DynamoDBBillingMode
+
+Set the DynamoDB billing mode for the API. One of "PROVISIONED" or "PAY_PER_REQUEST".
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.models['Post'].modelDDBTable.billingMode = 'PAY_PER_REQUEST'
+}
+```
+### Example - Configure DynamoDBModelTableProvisionedThroughput
+Override the default IOPS provisioned for each @model table
```

Comment:
```suggestion
Override the default IOPS provisioned for each `@model` table
```

encapsulate directive reference with an inline code block

Reply:
what does IOPS stand for? Can we include its name followed by the acronym, including a link to relevant docs?